docs/overview.md

```diff
@@ -0,0 +1,33 @@
+{% docs __overview__ %}
+
+## dbt-sythnea
+
+dbt-synthea is a project to demonstrate the power of dbt for building OMOP ETLs. Using [Synthea](https://synthetichealth.github.io/synthea/), it is possible to generate a synthetic patient data which can be transformed into the OMOP standard.
```

Comment:
maybe add a hyperlink on dbt as well, to dbt docs?

Reply:
Good shout